example/lib/password_benchmark.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:benchmark_harness/benchmark_harness.dart';
+import 'package:form_builder_validators/form_builder_validators.dart';
+import 'package:form_builder_validators/new_api_prototype.dart'
+    show req, password;
+
+import 'package:flutter/material.dart';
+// ignore_for_file: public_member_api_docs
+
+class ReqPasswordBenchmark extends BenchmarkBase {
+  const ReqPasswordBenchmark() : super("ReqPasswordBenchmark");
+  static void main() {
+    const ReqPasswordBenchmark().report();
+  }
+
+  @override
+  void run() {
+    final FormFieldValidator<String> v = req(password());
+    v('h');
+    v('he');
+    v('hel');
+    v('hell');
+    v('hello');
+    v('hello ');
+    v('hello W');
+    v('hello Wo');
+    v('hello Wor');
+    v('hello Worl');
+    v('hello World');
+    v('hello World!');
+    v('hello World!1');
+    v('hello World!12');
+    v('hello World!123');
+  }
+}
+
+// Benchmark for FormFieldValidators.password()
+class FormFieldValidatorsPasswordBenchmark extends BenchmarkBase {
+  const FormFieldValidatorsPasswordBenchmark()
+      : super("FormFieldValidatorsPasswordBenchmark");
+
+  static void main() {
+    const FormFieldValidatorsPasswordBenchmark().report();
+  }
+
+  @override
+  void run() {
+    // create the validator
+    final FormFieldValidator<String> v = FormBuilderValidators.password();
+    v('h');
+    v('he');
+    v('hel');
+    v('hell');
+    v('hello');
+    v('hello ');
+    v('hello W');
+    v('hello Wo');
+    v('hello Wor');
+    v('hello Worl');
+    v('hello World');
+    v('hello World!');
+    v('hello World!1');
+    v('hello World!12');
+    v('hello World!123');
+  }
+}
+
+void main() {
```

Comment:
Could create a benchmark of current API? To compare results

Reply:
I tried to create benchmarks but the problem is that the code need flutter libs (localization), which makes it difficult to use this benchmark lib. I will investigate better options or solutions. If you have one, please, share with me.

Reply:
Of. For now, leave this with only current benchmark